src/config/config.js

```diff
@@ -0,0 +1,28 @@
+import _ from "lodash";
+import configJson from "./config.json"
+
+let configuration = {};
+
+let configJsonOverride = null;
+
+
+if (process.env.CONFIG_OVERRIDE) {
+    try {
+        configJsonOverride = require(process.env.CONFIG_OVERRIDE);
+    } catch (e) {
+        // Do nothing
+    }
+}
+
+if (configJsonOverride) {
+    configuration = _.defaultsDeep(configJsonOverride, configJson);
+}
+
+if (process.env.LTI_BACKEND) {
+    configuration['lti_backend'] = process.env.LTI_BACKEND
+}
+
+
```

Comment:
and avoid extra blank lines; The Ultra team does some pretty tight linting and we may want to do that for this project as well